packages/api-gateway/__tests__/places/getPlacesFail.test.js

```diff
@@ -0,0 +1,16 @@
+const { fastify } = require('../../src/drivers/http/server');
```

Comment:
based on the comment below
```suggestion
const { getPlaces } = require('@booking-services/places');

const { fastify } = require('../../src/drivers/http/server');
```